ls1intum/Artemis
# `Exam mode`: Remove option to not include exercises into exam score

### Checklist
#### General


#### Client

### Motivation and Context
Currently, it is possible to specify that exam tasks should not be included in the score at all.
Following the issue [#4704](https://github.com/ls1intum/Artemis/issues/4704) this option has been removed.

### Description
- Modified the `jhi-included-in-overall-score-picker`, so that it now receives a boolean value if the option to not include the task in the score should not be displayed. The boolean value is equal to the boolean value `isExamMode`   (`{modeling;programming;text;quiz}-exercise-update.component.html` )
- Automatically assigning `INCLUDED_COMPLETELY` within the `ngOnInit` method, in case an exam task is imported, that is not included into the score. (`{modeling;programming;text;quiz}-exercise-update.component.ts` )

### Steps for Testing
Prerequisites:
- 1 Instructor
- 1 Programming Exercise, 1 Modeling Exercise, 1 Text-Exercise with points not included in the score

1. Log in to Artemis
2. Navigate to Course Administration -> Exam
3. Create a new Exam with Exercise Groups
4. Create a new Programming / Modeling / Quiz / Text / File-upload Exercise and makes sure that there is no selection option whether the task should not be included in the score. Only the options to include the exercise completely or as bonus should be visible & selectable (see Screenshot)
5. Import an existing Programming / Modeling / Text Exercise that is not included in the score, and makes sure that after importing the points are included in the score.

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage






### Screenshots


![image](https://user-images.githubusercontent.com/94070506/157733953-d63dadd4-8256-4d38-bcab-3bf1315d8bef.png)
